portal/src/main/webapp/js/api/cbioportal-datamanager.js

```diff
@@ -1144,6 +1147,7 @@ window.initDatamanager = function (genetic_profile_ids, oql_query, geneset_ids,
 		if (data_by_id[geneset_id][case_id].profile_data === null) {
 		    // set the initial value for this sample or patient
 		    data_by_id[geneset_id][case_id].profile_data = sample_score;
+		    data_by_id[geneset_id][case_id].na = false;
```

Comment:
oh I understand, sorry I didn't realize the way this was working. this should be right, then